int32 m1EncoderCount
int32 m2EncoderCount
float32 m1Amps
float32 m2Amps
float32 voltage